myst_nb/parser.py

```diff
@@ -142,15 +143,7 @@ def parse_block(src, start_line):
 
         elif nb_cell["cell_type"] == "code":
             # here we do nothing but store the cell as a custom token
-            block_tokens.append(
-                Token(
-                    "nb_code_cell",
-                    "",
-                    0,
-                    meta={"cell": nb_cell},
-                    map=[start_line, start_line],
-                )
-            )
+            block_tokens.extend(_parse_code_cell(nb_cell, start_line, parse_block))
```

Comment:
so here, could we just keep the old code the way it was, and then add a function that checks the outputs for markdown and, if there, appends it to `block_tokens` as a new markdown cell following the logic here: https://github.com/executablebooks/MyST-NB/pull/198/files#diff-a5ed38466bac00edd65797adb0d40849L136?

e.g. something like:

```python
remove_ixs = []
extra_markdown = []
for ii, output in enumerate(cell.outputs):
  if output.get("output_type") == "display_data" & "text/markdown" in output.get("data", []):
    extra_markdown.append(output["data"])
    remove_ixs.append(ii)
for ii in remove_ixs[::-1]:
    cell.outputs.pop(ii)
for markdown_cell in extra_markdown:
    block_tokens.extend(parse_block(markdown_cell , start_line))	
```

Reply:
I don't know how to get this into a transformer. every which way i tried, i could never use a toctree. having that parse_block function scoped in the parser seems to be collecting headings.

Reply:
i may be misunderstanding the comment.